src/mame/drivers/ddenlovr.cpp

```diff
@@ -9714,6 +9714,7 @@ MACHINE_START_MEMBER(ddenlovr_state,hanakanz)
 	uint8_t *rom = memregion("maincpu")->base();
 	membank("bank1")->configure_entries(0, 0x10, &rom[0x10000], 0x8000);
 	membank("bank2")->configure_entries(0, 0x10, &rom[0x90000], 0x1000);
+	subdevice<nvram_device>("nvram2")->set_base(&rom[0x90000], 0x1000 * 0x10);
```

Comment:
This can’t be right – you shouldn’t be using ROM regions for read/write memory.  If part of the stuff in the ROM region is actually default NVRAM contents, move it to a separate region and use it to initialise your NVRAM, and provide a view into that.

Reply:
the driver originally use that part of memory for banked RAM and I did not touch that to avoid introducing possible bugs.  It was probably a improper hack rather than actual ROM contents.

Do I need to change that?

Reply:
the driver originally uses that part of memory for banked RAM and I did not touch that, as I dont want to change unrelated stuff and introduce possible bugs. It was probably an improper hack rather than actual ROM contents.

eg:
https://github.com/mamedev/mame/blob/master/src/mame/drivers/ddenlovr.cpp#L11528

It allocates 0x90000+16*0x1000 bytes, which is rom region + banked ram (and no ROM is loaded into the banked ram part). Many games in this driver are implemented in this way.

Do I need to change all of these?